### PR TITLE
[CST] - Update CST V1 Components to V3 Components PR #3

### PR DESCRIPTION
**Pull request by pmclaren19**
## Summary

Updated the following components to use uswds
- `EvidenceWarning.jsx` for va-alert , also removed classsName that wasnt being used
- `WIP.jsx` for va-alert and va-telephone
- `AppealHelpSidebar.jsx` for va-telephone
- `Issues.jsx` for va-accordian
- `AdditionalEvidencePage.jsx` for va-loading-indicator
- `AdditionalEvidencePageOld.jsx` for va-loading-indicator
- `AskVAPage.jsx` for va-button and also updated va-checkbox to use uswds and updated the label text so that it looks better. Worked with @skylerschain to minimize the text and add a <p> above the checkbox.
- `DocumentRequestPage.jsx` for va-loading-indicator
- `StemClaimStatusPage.jsx` for va-loading-indicator
- `ClaimStatusApp.jsx` for va-loading-indicator

Updated the following components so that they use the default and uswds isnt set directly
- `UploadStatus.jsx` for va-button 
- `Expander.jsx` for va-button
- `DocumentsFiled.jsx` for va-pagination
- `FilesNeeded.jsx` for va-alert 
- `FilesOptional.jsx` for va-alert
- `ClosedClaimAlert.jsx` for va-alert
- `RecentActivity.jsx` for va-alert and va-pagination
- `AddFilesForm.jsx` for va-button, va-text-input, va-select, va-checkbox, and va-additional-info

Updated WhatWeAreDoing.jsx and ClaimCard.jsx to no longer use uswds=false since that is not an option on the card https://design.va.gov/storybook/?path=/docs/components-va-card--default

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#77095

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| EvidenceWarning va-alert  | Screenshot 2024-03-27 at 9 16 30 AM | Screenshot 2024-03-27 at 9 16 57 AM |
| WIP | Screenshot 2024-03-27 at 10 17 56 AM | Screenshot 2024-03-27 at 10 18 40 AM |
| AppealHelpSidebar va-telephone | Screenshot 2024-03-27 at 11 05 48 AM | Screenshot 2024-03-27 at 11 14 19 AM |
| Issues va-accordian closed |Screenshot 2024-03-27 at 2 31 04 PM| Screenshot 2024-03-27 at 2 31 55 PM|
| Issues va-accordian open | Screenshot 2024-03-27 at 2 31 17 PM | Screenshot 2024-03-27 at 2 32 00 PM |
| AdditionalEvidencePage va-loading-indicator | Screenshot 2024-03-28 at 12 54 10 PM | Screenshot 2024-03-28 at 12 54 37 PM |
| AskVAPage va-checkbox | Screenshot 2024-03-28 at 1 24 01 PM | Screenshot 2024-03-28 at 2 08 36 PM |
| ------- | ------ | ----- |

## What areas of the site does it impact?

Claim Status Tool

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/claims-status/containers/AskVAPage.jsx [hunk 1]
```diff
@@ -110,27 +110,27 @@ class AskVAPage extends React.Component {
                 </li>
                 <li>The date benefits will begin if we approve your claim</li>
               </ul>
-              <div className="usa-alert usa-alert-info background-color-only claims-alert">
```
Comment (jerekshoe) [inline src/applications/claims-status/containers/AskVAPage.jsx:113]: I'm guessing that removing this was also something discussed with design?

**Review by jerekshoe — APPROVED** [review]

**Review by samcoforma — APPROVED** [review]
LGTM

File: src/applications/claims-status/containers/AskVAPage.jsx (hunk 1 above)
Comment (pmclaren19) [reply src/applications/claims-status/containers/AskVAPage.jsx:113]: Yes we discussed :)

**Comment by pmclaren19** [comment]
Team decided not to merge this pr since it is the last day of our contract. We will merge next week if we get the contract, otherwise whoever picks up this contract will need to finish this work.

**Pull request merged**